lib/pnode/pnode.mli

```diff
@@ -14,7 +14,7 @@ to initialize a server that runs asynchronously. Returns
 a reference to the node. *)
 val init :
   ?preprocess:(Message.t -> Message.t) ->
-  msg_handler:(Message.t -> bytes) ->
+  msg_handler:(Message.t -> bytes * bytes option) ->
```

Comment:
I though a lot (too much) about this, I think making `msg_handler` returning a `bytes * bytes option` (where payload:bytes and payload_signature:bytes option) is the best option.
What do you think @rosalogia?